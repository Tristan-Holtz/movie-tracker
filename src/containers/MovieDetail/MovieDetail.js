import React from 'react';

export const MovieDetail = ({ selectedMovie }) => {
  const image = `https://image.tmdb.org/t/p/w300/${selectedMovie.poster_path}`

  return (
    <main>
      <img src={image} />
      <h2>{selectedMovie.title}</h2>
      <p>{selectedMovie.release_date}</p>
      <p>{selectedMovie.overview}</p>
    </main>
  )
}