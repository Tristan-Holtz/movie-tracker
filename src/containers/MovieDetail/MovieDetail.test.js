import React from 'react';
import { MovieDetail } from './MovieDetail';
import { shallow } from 'enzyme';

describe('MovieDetail', () => {
  it('should match a snapshot', () => {
    const mockMovie = {
      "genre_ids": [
          27,
          9648,
          878,
          53
      ],
      "id": 555974,
      "poster_path": "/4U7hpTK0XTQBKT5X60bKmJd05ha.jpg",
      "title": "The Invisible Man",
      "overview": "When Cecilia's abusive ex takes his own life and leaves her his fortune, she suspects his death was a hoax. As a series of coincidences turn lethal, Cecilia works to prove that she is being hunted by someone nobody can see.",
      "release_date": "2020-02-26"
    }
    const wrapper = shallow(<MovieDetail selectedMovie={mockMovie} />)
  
    expect(wrapper).toMatchSnapshot()
  })
})