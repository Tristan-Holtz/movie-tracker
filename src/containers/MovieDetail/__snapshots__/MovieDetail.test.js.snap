// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieDetail should match a snapshot 1`] = `
<main>
  <img
    src="https://image.tmdb.org/t/p/w300//4U7hpTK0XTQBKT5X60bKmJd05ha.jpg"
  />
  <h2>
    The Invisible Man
  </h2>
  <p>
    2020-02-26
  </p>
  <p>
    When Cecilia's abusive ex takes his own life and leaves her his fortune, she suspects his death was a hoax. As a series of coincidences turn lethal, Cecilia works to prove that she is being hunted by someone nobody can see.
  </p>
</main>
`;
